docs/source/introduction/configuring.rst

```diff
@@ -66,9 +66,12 @@ contain the following::
    [Paths]
 
    VISA library: /usr/lib/libvisa.so.7
+   dependency paths=/my/otherpath/;/my/otherpath2
```

Comment:
Since this is windows specific I would use a more descriptive name such as (I think using dll in the name will clarify the impact):
- dll paths
- dll extra paths
or similar

Reply:
Makes sense, I'll use dll_extra_paths